.github/workflows/teardown_urbanos.yml

```diff
@@ -37,8 +37,9 @@ jobs:
           mode: auto-approve-destroy
           commentOnPr: false
         env:
-          # ensure edits to these are added to the "teardown_urbanos" workflow as well
+          # ensure edits to these are added to the "deploy_urbanos" workflow as well
```

Comment:
still feel like this is still teardown_urbanos, due to still being in the teardown_urbanos yml?